Gemfile

```diff
@@ -17,7 +17,7 @@ gem "therubyracer", platforms: :ruby
 gem "uglifier", ">= 2.7.2"
 
 # views
-gem "active_link_to"
+gem "active_link_to", ">= 1.0.3"
```

Comment:
@slavakisel why we need to define here version condition here, probably updated version in `Gemfile.lock` should be enough?